src/components/App.js

```diff
@@ -69,11 +82,13 @@ const App = () => {
             onStatusChange={handleStatusChange}
             onUpdateText={handleUpdateTodo}
             onDelete={handleDeleteTodo}
+            onClickDetailsTodo={handleClickDetailsTodo}
           >
             {todo.text}
           </Todo>
         ))}
       </TodoList>
+      {isModalShow && <Modal item={todoModal} onCloseModal={handleClickDetailsTodo} />}
```

Comment:
Acá debería llamarse según la convención `handleCloseModal`

```suggestion
      {isModalShow && <Modal item={todoModal} onCloseModal={handleCloseModal} />}
```

Reply:
Pablo ya realice todas las modificaciones, ahora lo subo para que lo veas y nos comentes. Por error comence a resolver las conversaciones de abajo hacia arriba asi que el comentario queda en el primero, perdon